oteapi_asmod/strategies/parse.py

```diff
@@ -63,24 +68,26 @@ def parse(self, session: "Optional[Dict[str, Any]]" = None) -> "Dict[str, Any]":
             session: A session-specific dictionary context.
 
         Returns:
-            Dictionary of key/value-pairs to be stored in the sessions-specific
-            dictionary context.
+            ase.Atoms object in cache.
 
         """
         model = AtomisticParseDataModel(
-            **self.resource_config.configuration, extra=Extra.ignore
+            **self.parse_config.configuration, extra=Extra.ignore
```

Comment:
I think that the `extra` keyword can be removed. Casper can confirm.
```suggestion
            **self.parse_config.configuration
```